pkg/hubble/metrics/api/context_test.go

```diff
@@ -514,26 +514,24 @@ func TestParseGetLabelValues(t *testing.T) {
 		TrafficDirection: pb.TrafficDirection_INGRESS,
 	}
 	assert.EqualValues(t,
-		mustGetLabelValues(opts, flow),
 		[]string{
-			// source_ip, source_pod, source_namespace, source_workload, source_workload_kind , source_app
```

Comment:
I think these comments were somewhat useful given it's just a slice of strings

Reply:
Sure, I'll restore them (it seems that `--fix` is not smart enough to preserve the comments :disappointed:.